### PR TITLE
Added Capture request and response

### PR DESCRIPTION
**Pull request by Feijs**

### CODE REVIEW

File: src/Message/CaptureRequest.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+final class CaptureRequest extends AbstractRequest
+{
+    use ItemDataTrait;
+
+    /**
+     * @inheritDoc
+     */
+    public function getData()
+    {
+        $this->validate('transactionReference', 'amount');
```
Comment (fred-jan) [inline src/Message/CaptureRequest.php:16]: Perhaps you could add the `items` key to the `validate` call?

Comment (Feijs) [reply src/Message/CaptureRequest.php:16]: They are not required for this action

Comment (Feijs) [reply src/Message/CaptureRequest.php:16]: I could however add `order_lines` to the output conditionally, instead of passing an empty array

Comment (fred-jan) [reply src/Message/CaptureRequest.php:16]: Then the `getItemData` call should be made conditional i guess?

**Pull request merged**